website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -111,6 +112,10 @@ var Range = ace.require('ace/range').Range;
             win: 'Ctrl-Y|Ctrl-Shift-Z',
             mac: 'Command-Y|Command-Shift-Z',
         },
+        help: {

```

Comment:
Is this supposed to open help when Cmd+U is pressed? If so, it's not working for me locally; it looks like Cmd+U is already mapped to create an unordered list.

We can probably remove this.